src/main/java/picard/vcf/LiftoverVcf.java

```diff
@@ -250,7 +253,10 @@
     );
 
     private VariantContextWriter rejects;
+    /** the output VariantContextWriter */
+    private VariantContextWriter accept;
```

Comment:
For consistency, can you rename this `accepts` (with `s`) or change this and `rejects` to `acceptedRecords` and `rejectedRecords` ?